Content.Server/Shuttles/Systems/ShuttleConsoleSystem.cs

```diff
@@ -271,9 +271,11 @@ private void UpdateState(EntityUid consoleUid, List<DockingInterfaceState>? dock
             ftlTime = _timing.CurTime + TimeSpan.FromSeconds(shuttleFtl.Accumulator);
         }
 
-        // Mass too large
+        // Is Mass of grid too large?
+        // OR is intended to be a Spaceship
+        // The 1000 mass limit exists as abstract divide between which grid "is a shuttle" and which "is a station"
         if (entity != null && shuttleGridUid != null &&
-            (!TryComp<PhysicsComponent>(shuttleGridUid, out var shuttleBody) || shuttleBody.Mass < 1000f))
+            (!TryComp<PhysicsComponent>(shuttleGridUid, out var shuttleBody) || HasComp<FTLWhitelistComponent>(shuttleGridUid) || shuttleBody.Mass < 1000f))
```

Comment:
If you could surround your change with a block like the one found [here](
https://github.com/Nyanotrasen/Nyanotrasen/wiki/Contributing-Guidelines#changing-upstream-code) which gives an explanation or rationale for the change, then we should be good. It should encapsulate all changed lines.

Reply:
I hope the explanation is understandable.